app/views/items/new.html.erb

```diff
@@ -0,0 +1,15 @@
+<h2>Add New Item</h2>
+
+<%= form_for @item do |f| %>
+  <%= f.label :name %>
+  <%= f.text_field :name%>
+  <%= f.label :description %>
+  <%= f.text_area :description %>
+  <%= f.label :category %>
+  <%= select(:item, :category_id, Category.all.map { |cat| [cat.name, cat.id] } ) %>
```

Comment:
Possible refactor to remove call to database(Category) in view, pass in category names/ids from controller or method on model. (Waffle card created)